02 - Community Expansions/02.01 Plugins by Category/Third-Party Integration Plugins.md

```diff
@@ -7,9 +7,9 @@ publish: true
 ---
 
 
-# Integration
+# Third-Party Integration Plugins
 
-Intergate with other services or applications
+Plugins to intergate with other services or applications.
```

Comment:
I think this category should differentiate between plugins that add one specific service to Obsidian (e.g. Readwise) and plugins that enable you do generally integrate third party services to Obsidian (e.g. URI Commands), since they are similar, but distinct use cases.

Reply:
Opened #61 for this